packages/web/src/application/test/zoom/zoom-provider.test.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/react';
+import application from '@application';
+
+import {
+  renderWithProvider,
+  renderWithProviderAndCustomConsumer,
+  getElementByRegex,
+  getContextInitialData,
+  mockDataResponse,
+  mockEmptyDataResponse,
+} from './utils';
+import { ZoomContext, ZoomContextType } from '../../../modules/zoom/context';
+
+beforeAll(() => application.init());
+
+const { fixtures } = application;
+
+describe('Zoom Provider', () => {
+  it('ZoomContext should have correct initial values', async () => {
+    const { student: user } = fixtures.users;
+    const { requests } = application;
+
+    requests.zoomSignature = mockEmptyDataResponse();
+    requests.zoomAuthorize = mockEmptyDataResponse();
+
+    const initialContext = getContextInitialData(user);
+
+    renderWithProvider(
+      <ZoomContext.Consumer>
+        {(value: ZoomContextType) =>
+          Object.keys(value).map(key => (
+            <span key={key}>
+              {`${key.toString()}: ${value[
+                key as keyof ZoomContextType
+              ]?.toString()}`}
+            </span>
+          ))
+        }
+      </ZoomContext.Consumer>,
+      { ...initialContext, user },
+    );
+
+    expect(await getElementByRegex(/userSignature/)).toBe(
+      'userSignature: undefined',
```

Comment:
Not sure because I am feeding the DOM with the context values to be able to test it, so I can't get it in that format.

` {(value: ZoomContextType) =>
          Object.keys(value).map(key => (
            <span key={key}>
              {`${key.toString()}: ${value[
                key as keyof ZoomContextType
              ]?.toString()}`}
            </span>
          ))
        }`

Reply:
Which property wasn't stringifiable?

Reply:
zoomSDKElementRef: jest.fn()